api/infrastructure/parallelcluster-api.yaml

```diff
@@ -610,6 +621,20 @@ Resources:
                 iam:AWSServiceName:
                   - fsx.amazonaws.com
                   - s3.data-source.lustre.fsx.amazonaws.com
+          - Action:
+              - iam:CreateServiceLinkedRole
+              - iam:AttachRolePolicy
+              - iam:PutRolePolicy
+            Resource: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/aws-service-role/s3.data-source.lustre.fsx.amazonaws.com/*
+            Effect: Allow
+          - Action:
+              - s3:Get*
+              - s3:List*
+              - s3:PutObject
+            Resource: '*'
+            Effect: Allow
+            Condition: !If
```

Comment:
Isn't enough to write `Condition: EnableS3FullAccessCondition`?

Reply:
Changed with the actual condition